cdap-ui/app/tracker/templates/main.html

```diff
@@ -15,17 +15,12 @@
 -->
 
 <div class="tracker-content">
-  <div class="tracker-top-search"
-       ng-if="!$state.params.iframe">
-    <div class="tracker-search-bar">
-      <div class="form-group has-feedback" ng-keypress="MainController.search($event)">
-        <input type="text"
-               class="form-control"
-               placeholder="Search datasets"
-               ng-model="MainController.searchQuery">
-        <span class="fa fa-search form-control-feedback"></span>
-      </div>
-    </div>
-    <cask-resource-center-button></cask-resource-center-button>
+  <div class="form-group has-feedback tracker-search" ng-keypress="MainController.search($event)">
+    <input type="text"
+           class="form-control"
+           placeholder="Search datasets"
+           ng-model="MainController.searchQuery"
+           pattern="^(?![*]).*$">
```

Comment:
I know this forbids the user from searching for `*`, but what's the reason behind that again?